src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -1851,6 +1947,7 @@ private void SaveCore(ModelSaveContext ctx)
             _host.CheckDecode(valuePrev <= 1);
         }
 
+        /// <summary> Given a classifier output, produce the probability.</summary>
```

Comment:
><summary> [](start = 12, length = 9)

break

Reply:
It fits the line, why it need break?

---
In reply to: [258187676](https://github.com/dotnet/machinelearning/pull/2601#discussion_r258187676) [](ancestors = 258187676)

Reply:
I thought the style was to break the xml tags onto their own lines, but I could be mistaken.

---
In reply to: [258188999](https://github.com/dotnet/machinelearning/pull/2601#discussion_r258188999) [](ancestors = 258188999,258187676)